src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -1096,162 +1095,153 @@ def initialize_build(*args, **kwargs):
 
     def report(self, export=False):
         var_dict = {}
+        var_dict["Plant Cost Units"] = str(pyunits.get_units(self.total_plant_cost))
 
         if hasattr(self, "total_plant_cost"):
-            var_dict["Total Plant Cost [$MM]"] = value(self.total_plant_cost)
+            var_dict["Total Plant Cost"] = value(self.total_plant_cost)
```

Comment:
As a passing note (not for this PR), you can use `as_quantity` (from Pyomo) to get a `pint` Quantity instead (which contains units, and has some useful features for automatically converting the value to a set of display units).